packages/flutter/lib/src/rendering/sliver_fixed_extent_list.dart

```diff
@@ -88,7 +88,7 @@ abstract class RenderSliverFixedExtentBoxAdaptor extends RenderSliverMultiBoxAda
     if (itemExtent > 0.0) {
       final double actual = scrollOffset / itemExtent - 1;
       final int round = actual.round();
-      if (_isWithinPrecisionErrorTolerance(actual, round)) {
+      if ((actual - round).abs() < precisionErrorTolerance) {
```

Comment:
Changes in this file should also have a test. It appears that the test here only covers the changes in `scroll_activity.dart`.